### DISCUSSION TITLE
Row Filters

### DISCUSSION THREAD

**Discussion by Niyanthan** [fbaligand/kibana-enhanced-table#220] [Q&A]

**Comment by fbaligand** [comment]
Well, given your initial description, you can do this with this expression:
`col1 > 5000 and col2 == "Low"`

Important note: col0 is the first column and col1 is the second column. Column indices are 0-based. 

By the way, you can found full expression documentation here:
https://github.com/silentmatt/expr-eval#expression-syntax

Fabien
